composer.json

```diff
@@ -44,7 +44,8 @@
         "league/csv": "~9.1",
         "nesbot/carbon": "^2.0",
         "laravel/framework": "~6.0",
-        "laravel/tinker": "~2.0"
+        "laravel/tinker": "~2.0",
+        "composer/semver": "^3.2"
```

Comment:
This should probably just be done in the system module itself if it's not being used anywhere else in the application.

Reply:
https://github.com/wintercms/wn-system-module/pull/2

I've now closed this PR